sdk/aqueduct/aqueduct_client.py

```diff
@@ -246,6 +248,26 @@ def list_flows(self) -> List[Dict[str, str]]:
             workflow_resp.to_readable_dict() for workflow_resp in self._api_client.list_workflows()
         ]
 
+    def get_workflow_writes(
+        self, flow_id: Union[str, uuid.UUID]
+    ) -> DefaultDict[uuid.UUID, List[WrittenObject]]:
+        """Get everything written to by the flow identified by the flow id (via `.save`).
+
+        Returns:
+            A dictionary mapping the integration id to the list of table names/storage path.
+        """
+        flow_id = parse_user_supplied_id(flow_id)
+
+        if all(uuid.UUID(flow_id) != workflow.id for workflow in self._api_client.list_workflows()):
+            raise InvalidUserArgumentException("Unable to find a flow with id %s" % flow_id)
+
+        workflow_writes = self._api_client.get_workflow_writes(flow_id).object_details
+        writes_mapping = defaultdict(list)
+        for item in workflow_writes:
+            written_object = WrittenObject(item.object_name, item.update_mode)
+            writes_mapping[item.integration_id].append(written_object)
+        return writes_mapping
```

Comment:
I feel like `save` is a better term we should use throughout, since it corresponds with our `.save()` method. "writes" has a more generic connotation.